lib/Centreon/Object/Relation/Host/Template/Host.php

```diff
@@ -70,6 +70,38 @@ public function insert($fkey, $skey = null)
         $this->db->query($sql, array($fkey, $skey, $order));
     }
 
+    /**
+     * Delete host template / host relation and linked services
+     *
+     * @param int $fkey
+     * @param int $skey
+     * @return void
+     */
+    public function delete(int $fkey, int $skey = null): void
+    {
+        $this->db->beginTransaction();
+        try {
+            parent::delete($fkey, $skey);
+            // Delete linked services :
+            // First get service IDs
+            $sql = 'SELECT service_service_id FROM host_service_relation WHERE host_host_id = ?';
```

Comment:
```suggestion
            $linkedServices = $this->getResult(
                'SELECT service_service_id FROM host_service_relation WHERE host_host_id = ?',
                [(int)$fkey]
            );
```
remove the `$sql` variable